package_manifest.yaml

```diff
@@ -1087,6 +1088,7 @@ plugin_nonscl_packages:
     ansiblerole-satellite-receptor-installer: {}
     puppet-foreman_scap_client: {}
     puppetlabs-stdlib: {}
+    rubygem-smart_proxy_acd: {}
```

Comment:
this should be in `plugin_scl_packages` -- does `add_gem_package` need an update here?